### PR TITLE
feat: silkscreen text

### LINKED ISSUE #60: implement silkscreen text

### PR DESCRIPTION
**Pull request by Abse2001**
/claim #60
/closes #60

### CODE REVIEW

**Comment by seveibar** [comment]
Looks great! What’s up with the”8” and “e”? Why does it need special handling?

**Comment by Abse2001** [comment]
> Looks great! What’s up with the”8” and “e”? Why does it need special handling?

IDK why but when the last point in 8 tries to close on the first point the holes fill and it starts to look weird and the e with the hole to

File: src/soup-to-3d/index.ts [hunk 1]
```diff
@@ -219,9 +221,118 @@ export const createBoardGeomFromSoup = (soup: AnySoupElement[]): Geom3[] => {
       pcb_plated_hole_id: "",
     })
   }
+  // Add silkscreen text
+  const silkscreenGeoms: Geom3[] = []
+  for (const silkscreenText of silkscreenTexts) {
+    // Generate 2D text outlines
```
Comment (seveibar) [inline src/soup-to-3d/index.ts:227]: Can you isolate the text vector outline handling stuff into its own function? Like createGeomForSilkscreenText and give it its own file?

**Comment by Abse2001** [comment]
without handling 

image

**Review by seveibar — APPROVED** [review]
AMAZING

**Pull request merged**